common/types.go

```diff
@@ -466,17 +463,34 @@ func (loc Location) CommonDom(cmp Location) Location {
 }
 
 func (l Location) ContainsAddress(a Address) bool {
-	// ContainAddress can only be called for a zone chain
+	// Ensure the Location has content
+	if len(l) < 1 {
```

Comment:
I think this length check should be `if len(l) < 2`, we cannot have address in region as well and region location is of length 1

Reply:
Also the thing I don't understand is why are we changing the way we were indexing Location? Wasn't having a byte array like before fine?